core-tests/shared/src/test/scala/zio/ExitSpec.scala

```diff
@@ -11,6 +11,7 @@ class ExitSpec extends Specification with ScalaCheck {
       `Cause#died` and `Cause#stripFailures` are consistent $e1
       `Cause.equals` is symmetric $e2
       `Cause.equals` and `Cause.hashCode` satisfy the contract $e3
+      `Cause#untraced` satisfy the contract $e8
```

Comment:
You may reword this as as `Cause#untraced removes all traces`, note that "the contract" on the line above refers to the contracts of .equals() and .hashCode() as defined by Java Specification; there's no such widely known contract attached to `untraced`, so the current wording may seem somewhat off

Reply:
Sure.